templates/cdhpages/blocks/tile.html

```diff
@@ -1,102 +1,101 @@
-            {% else %}
-                {# placeholder image #}
-            {% endif %}
-            <div>{{ internal_page.specific.short_title|default:internal_page.specific.title }}</div>
-            </div>{{ internal_page.specific.person.current_title }}</div>
-            <a href="{{ internal_page.url }}">{{ internal_page.url }}</a>
-        {% endif %}
+            {# Internal page deets #}
+            {% if tile.block.name == 'internal_page_tile' %}
 
-        {% if internal_page.specific.page_type == 'Project' %} 
-            {% if internal_page.specific.feed_image %}
-                {% image internal_page.specific.feed_image fill-600x348 format-webp as image %}
-                <img src="{{ image.url }}" alt="{{ image.title }}"/>
-            {% elif internal_page.specific.hero_image %}
-                {% image internal_page.specific.hero_image fill-600x348 format-webp as image %}
-                <img src="{{ image.url }}" alt="{{ image.title }}"/>
-            {% else %}
-                {# placeholder image #}
-            {% endif %}
-            <div>{{ internal_page.specific.short_title|default:internal_page.specific.title }}</div>
-            <div>{{ internal_page.specific.short_description|default:internal_page.specific.description|richtext }}</div>
-            {% if internal_page.specific.cdh_built %}<div class="cdh-built">Built by CDH</div>{% endif %}
-            <a href="{{ internal_page.url }}">{{ internal_page.url }}</a>
-        {% endif %}
+                {{ internal_page.specific.description|richtext }}
+
+                {# Extra event-specific deets #}
+                {% if internal_page.specific.page_type == 'Event' %}
+                    
+                    {# Complex logic to show differenty formats depending on if the event spans multiple days (or months!) #}
+                    {% with start_day=internal_page.specific.start_time|date:"j" %}
+                    {% with start_month=internal_page.specific.start_time|date:"M" %}
+                    {% with end_day=internal_page.specific.end_time|date:"j" %}
+                    {% with end_month=internal_page.specific.end_time|date:"M" %}
+                        <div class="tile__event-date">
+                            {% if start_day == end_day and start_month == end_month %}
+                                {{ start_day }} {{ start_month }} {{ internal_page.specific.start_time|time:"g:iA" }}–{{ internal_page.specific.end_time|time:"g:iA" }}
+                            {% elif start_month == end_month %}
+                                {{ start_day }}–{{ end_day }} {{ start_month }}
+                            {% else %}
+                                {{ start_day }} {{ start_month }}–{{ end_day }} {{ end_month }}
+                            {% endif %}
+                        </div>
+                    {% endwith %}
+                    {% endwith %}
+                    {% endwith %}
+                    {% endwith %}
```

Comment:
I got AI to help with this. Maybe it could be better, I dunno? It needs various different formats depending on whether the event is a single day (in which case show the start/end time), multiple day, or spanning multiple months.

Reply:
@sarahframe / @haydngreatnews